options.go

```diff
@@ -5,10 +5,11 @@
 package temporalite
 
 import (
+	"github.com/DataDog/temporalite/internal/liteconfig"
 	"go.temporal.io/server/common/log"
 	"go.temporal.io/server/temporal"
 
-	"github.com/DataDog/temporalite/internal/liteconfig"
+	temporalconfig "go.temporal.io/server/common/config"
```

Comment:
Nit: no need to alias this import. It'd also be nice to keep the `go.temporal.io/server` imports grouped together.